pkg/harvester/l10n/en-us.yaml

```diff
@@ -307,6 +307,7 @@ harvester:
     console: Console
     label: Hosts
     inconsistentIP: "Host IP is inconsistent, current IP: { currentIP }, initial IP: { initIP }"
+    noConsoleUrl: 'Console URL is empty'
```

Comment:
```suggestion
    noConsoleUrl: 'Console URL not specified.'
```

Reply:
Done modified. 
We can easily update the harvester docs in `config/doc-links.js`. Just let us know the new product docs is online available.